project/SparkBuild.scala

```diff
@@ -703,32 +703,24 @@ object SparkConnect {
 }
 
 object SparkProtobuf {
-
   import BuildCommons.protoVersion
 
-  private val shadePrefix = "org.sparkproject.spark-protobuf"
-  val shadeJar = taskKey[Unit]("Shade the Jars")
-
   lazy val settings = Seq(
     // Setting version for the protobuf compiler. This has to be propagated to every sub-project
     // even if the project is not using it.
     PB.protocVersion := BuildCommons.protoVersion,
 
     // For some reason the resolution from the imported Maven build does not work for some
     // of these dependendencies that we need to shade later on.
-    libraryDependencies ++= Seq(
-      "com.google.protobuf" % "protobuf-java"        % protoVersion % "protobuf"
-    ),
+    libraryDependencies += "com.google.protobuf" % "protobuf-java" % protoVersion % "protobuf",
 
-    dependencyOverrides ++= Seq(
-      "com.google.protobuf" % "protobuf-java"        % protoVersion
-    ),
+    dependencyOverrides += "com.google.protobuf" % "protobuf-java" % protoVersion,
 
     (Compile / PB.targets) := Seq(
-      PB.gens.java                -> (Compile / sourceManaged).value,
+      PB.gens.java -> (Compile / sourceManaged).value,
     ),
 
-    (assembly / test) := false,
+    (assembly / test) := { },
```

Comment:
just for my own education, is this the same: `false` -> `{ }`? 

looks like a bit magic

Reply:
Yeah, they are same. I believe `{ }` is much more common (googled a bit)